docs/02-app/01-building-your-application/03-rendering/03-edge-and-nodejs-runtimes.mdx

```diff
@@ -27,7 +27,9 @@ There are many considerations to make when choosing a runtime. This table shows
 | Security                                                                                                                                            | Normal | High       | High             |
 | Latency                                                                                                                                             | Normal | Low        | Lowest           |
 | npm Packages                                                                                                                                        | All    | All        | A smaller subset |
-| Revalidation                                                                                                                                        | Yes    | Yes        | No               |
+| [Static Rendering](/docs/app/building-your-application/rendering/static-and-dynamic#static-rendering-default)                                       | Yes    | Yes        | No               |
+| [Dynamic Rendering](/docs/app/building-your-application/rendering/static-and-dynamic#dynamic-rendering)                                             | Yes    | Yes        | Yes              |
+| [Data Revalidation w/ `fetch`](/docs/app/building-your-application/data-fetching/fetching-caching-and-revalidating#revalidating-data)               | Yes    | Yes        | Yes              |
```

Comment:
I thought this PR was supposed to say Edge "No" for revalidation but I see "Yes" here <picture data-single-emoji=":thonking2:" title=":thonking2:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/thonking2/63d309b4d143d783.gif" alt=":thonking2:" align="absmiddle"></picture>